### ISSUE TITLE
[Bug] Rampant overgrowth of iron ore

### ISSUE THREAD

**Issue by Sunconure11** [SuperMartijn642/OreGrowth#2]
**Version Info**
- Minecraft, Quilt 1.19.2
- Ore Growth, 1.0.1

**What mod loader are you using?**: Quilt

**Description of the Bug**
Iron... should not be doing this. @SuperMartijn642 

**Steps to Reproduce**
Go into a world

**Screenshots**
![image](https://github.com/SuperMartijn642/OreGrowth/assets/5614902/0218a3ca-82ec-44c9-b10a-45219128fd2d)

**Comment by Sunconure11** [comment]
![image](https://github.com/SuperMartijn642/OreGrowth/assets/5614902/8037615c-f7cc-45bd-adf4-0540dd1ac00c)

Good lord

**Comment by SuperMartijn642** [comment]
Could you explain a bit 😅 I see there's **a lot** of crystals, but apart from that I am not exactly sure what I'm looking at. Is that like a void world or something. Or are you in spectator, I don't see any caves.

**Comment by Sunconure11** [comment]
That's the limbo dimension from Dimensional Doors.

**Comment by SuperMartijn642** [comment]
Are the crystals just growing in the air though? Or are there blocks I don't see. I have never used Dimensional Doors, so not exactly sure what the dimension is like 😅

**Comment by Sunconure11** [comment]
They're all over. And not just that dimension. Even the overworld is affected.

**Comment by SuperMartijn642** [comment]
But just in the air or are they growing on ore blocks?

**Comment by Sunconure11** [comment]
Both.

**Comment by Sunconure11** [comment]
![image](https://github.com/SuperMartijn642/OreGrowth/assets/5614902/12bf4770-41e5-4b1e-b2c3-28b0014e037c)

**Comment by SuperMartijn642** [comment]
Ah thank you, that is very odd 🤔

**Comment by SuperMartijn642** [comment]
For anyone looking at this, the issue was due to a custom recipe by @Sunconure11 which happened to have an invalid base block. The block registry returns the air block for missing blocks, thus the crystals ended up growing on air.

I have now fixed the check for the base block to make sure a block with the given id is actually registered, to make sure a scenario like this will give a proper error.
